src/NuGet.Core/NuGet.PackageManagement/BuildIntegration/DependencyGraphRestoreUtility.cs

```diff
@@ -141,7 +141,7 @@ public static class DependencyGraphRestoreUtility
                 cacheContextModifier(sourceCacheContext);
 
                 // Settings passed here will be used to populate the restore requests.
-                var restoreContext = GetRestoreContext(context, providerCache, sourceCacheContext, sources, dgFile, false); // We don't force in preview 
+                var restoreContext = GetRestoreContext(context, providerCache, sourceCacheContext, sources, dgFile, forceRestore : true);
```

Comment:
just curious, wouldn't there be major perf loss too with no-op now? can we scope this down for legacy scenarios?